server/examples/go/hello/submit_survey.go

```diff
@@ -0,0 +1,36 @@
+package hello
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/apps/mmclient"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func SubmitSurvey(c *apps.Call) *apps.CallResponse {
+	location := strings.Split(string(c.Context.Location), "/")
+	if len(location) == 0 {
+		return &apps.CallResponse{
+			Type:      apps.CallResponseTypeError,
+			ErrorText: "Wrong location.",
+		}
+	}
+	selected := location[len(location)-1]
+	if selected == "button" {
+		bot := mmclient.AsBot(c.Context)
+		p := &model.Post{
+			Id:      c.Context.PostID,
+			Message: "The survey will not be sent",
+		}
+		_, err := bot.UpdatePost(c.Context.PostID, p)
+		fmt.Println(err)
```

Comment:
do we need these?

Reply:
Nope, I added those when I was debugging and forgot to remove them :facepalm: